test/kubetest/k8s.go

```diff
@@ -540,6 +548,22 @@ func (k8s *K8s) describePod(pod *v1.Pod) []v1.Event {
 	return result
 }
 
+func (k8s *K8s) buildNSMConfigMap() *v1.ConfigMap {
+	return &v1.ConfigMap{
+		TypeMeta: metaV1.TypeMeta{
+			Kind:       "ConfigMap",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metaV1.ObjectMeta{
+			Name:      "nsm-config",
+			Namespace: k8s.GetK8sNamespace(),
+		},
+		Data: map[string]string{
+			"excluded_prefixes.yaml": "",
```

Comment:
Can we use value from constant `PrefixesFile` here?

Reply:
- [ ] use value from constant PrefixesFile